R/stan-likelihood.R

```diff
@@ -889,6 +889,25 @@ stan_log_lik_hurdle_cumulative <- function(bterms, resp = "", mix = "",
   sdist(lpdf, p$mu, p$hu, p$disc, p$thres, p$Jthres)
 }
 
+stan_log_lik_mixcure_lognormal <- function(bterms, resp = "", mix = "", ...) {
+  p <- stan_log_lik_dpars(bterms, TRUE, resp, mix)
+  usc_logit <- stan_log_lik_dpar_usc_logit("inc", bterms)
+  lpdf <- paste0("mixcure_lognormal", usc_logit)
+  sdist(lpdf, p$mu, p$sigma, p$inc)
+}
+
+stan_log_lik_mixcure_weibull <- function(bterms, resp = "", mix = "", ...) {
```

Comment:
It seems like `reqn` should always be true given the stan code of this model. If true, should the code below be changed or simplified?